concepts/05 Widgets/Slider/05 Customize Widget Appearance.md

```diff
@@ -41,6 +239,83 @@ To specify whether or not the part of the scale from the beginning to the slider
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-slider
+        [showRange]="showRange"
+    </dx-slider>
+
+    <!--TypeScript-->
+    import { DxSliderModule } from "devextreme-angular";
+    // ...
+    export class AppComponent {
+        showRange = false;
+    }
+    @NgModule({
+        imports: [
+            // ...
+            DxSliderModule
+        ],
+        // ...
+    })
+
+##### Vue
+
+    <template>
+        <DxSlider
+            :showRange="showRange"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxSlider } from 'devextreme-vue/slider';
+
+    export default {
+        components: {
+            DxSlider
+        },
+        data() {
+            return {
+                showRange: false
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Slider } from 'devextreme-react/slider';
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+
+            this.state = {
+                showRange: false
```

Comment:
Same thing here - use state only for those properties that are going to be changed.

Reply:
fixed